liquidprompt

```diff
@@ -2976,6 +3008,11 @@ __lp_set_prompt() {
     fi
 }
 
+# If we want this done with a timer for debug purposes, and if we're using bash-preexec, then this must be a function
+__lp_debug_timed_lp_set_prompt() {
+    time __lp_set_prompt
+}
+
```

Comment:
I would rather see this an inline function. Define it as part of the block of line 3100, then use it.

```bash
            (( LP_DEBUG_TIME )) {
                __lp_debug_timed_lp_set_prompt() {
                    time __lp_set_prompt
                }
                precmd_functions+=(__lp_debug_timed_lp_set_prompt)
            }
```

Reply:
Done in 26d2dd9